test/Microsoft.AspNetCore.Mvc.Core.Test/Internal/ControllerActionInvokerTest.cs

```diff
@@ -155,7 +155,7 @@ public class ControllerActionInvokerTest
                 .Callback<ExceptionContext>(context =>
                 {
                     filter2.ToString();
-                    context.Exception = null;
+                    context.ExceptionHandled = true;
```

Comment:
We should continue to support short circuiting by setting the exception to null. There should be tests for both of these cases